docs/feed-changelog.md

```diff
@@ -0,0 +1,12921 @@
+# Changes to NI Linux RT Package Feeds
+
+## Description
+
+This documents the full changes to the core/ and extra/ feeds for each version release.
+
+The changes are too long to keep in the [Changelog](/CHANGELOG.md) itself, so they are kept here.
+
+Generate them by:
+    1. adding a line to ./docs/feed-changelog.json: `./scripts/diff-pkg-changes.py <branch of old version> <branch of new version> >> ./docs/feed-changelog.json`
+    2. updating the markdown with: `./scripts/emit-feed-changelog-md.py < ./docs/feed-changelog.json > ./docs/feed-changelog.md`
+
+## Feed Changes from Version 8.11 to Version 9.0
+
+### Changes to core/
+
+* added:
```

Comment:
I don't think this output is correct. :/ I see (a) way too many packages and (b) many packages listed here which should have been in both the 8.11 and 9.0 feed.

Reply:
I agree it seems like a lot. I'm looking into it, but I haven't had any luck finding an issue. Can you give an example of a package listed that should be in both but is showing as not?

I figured the increase from earlier code versions was from switching from just using http://download.ni.com/ni-linux-rt/feeds/2021.8/x64/main/all to also including /main/x64 and /main/core2-64, but it definitely could be an issue instead.

Reply:
One thing that seems to affect it is that some packages are marked as "removed" when they change architecture. That doesn't seem to happen with "added," but it could

Reply:
Yeah; after a more careful review: this list does actually seem accurate. A lot of packages genuinely got added or removed between 8.X (sumo) and 9.0 (hardknott). And many more changed from the extra/ feed to core/ - which causes them to show up under both added and removed.

So I guess this is fine.